tests/integration/template_debug.cfg

```diff
@@ -5,53 +5,57 @@
 # 4. Once it's working as expected, copy important changes to template_complete_run.cfg.
 
 [default]
-case = v2.LR.historical_0201
+case = v2.LR.BGC-LNDATM.CONTRL
 constraint = "#expand constraint#"
 dry_run = "#expand dry_run#"
 environment_commands = "#expand environment_commands#"
-input = "#expand user_input#/E3SMv2/v2.LR.historical_0201"
+#input = "#expand user_input#/E3SMv2/v2.LR.historical_0201"
+input = "/lcrc/group/e3sm/ac.sfeng1/E3SM_Simulations/20221127.v2.LR.BGC-LNDATM.CONTRL.ne30pg2_r05_EC30to60E2r2.chrysalis"
 input_subdir = archive/atm/hist
-mapping_file = "map_ne30pg2_to_cmip6_180x360_aave.20200201.nc"
+mapping_file = "map_r05_to_cmip6_180x360_aave.20231110.nc"
 # To run this test, edit `output` and `www` in this file, along with `actual_images_dir` in test_complete_run.py
-output = "#expand user_output#zppy_test_debug_output/#expand unique_id#/v2.LR.historical_0201"
+output = "#expand user_output#zppy_test_debug_output/#expand unique_id#/20221127.v2.LR.BGC-LNDATM.CONTRL.ne30pg2_r05_EC30to60E2r2.chrysalis"
 partition = "#expand partition_short#"
 qos = "#expand qos_short#"
-www = "#expand user_www#zppy_debug_www/#expand unique_id#"
+vars = "TS,FSNT,FLNT,TOTSOMC,TOTECOSYSC,NEE,TOTVEGC"
+www = "#expand user_www#zppy_test_debug_www/#expand unique_id#"
 
 [ts]
 active = True
+e3sm_to_cmip_environment_commands = "#expand e3sm_to_cmip_environment_commands#"
+vars = "TS,FSNT,FLNT,TOTSOMC,TOTECOSYSC,NEE,TOTVEGC"
```

Comment:
@forsyth2  is it okay not to specify vars here under [ts]

Reply:
It shouldn't have to be, since it's specified under `[default]` too, but I think I added it there because I ran into a problem during testing. I'm not sure if that was resolved though, so might be worth removing it to see. I can try that too.

(Also, I see this is the original debug file. I think `debug3` is probably more useful.)

Reply:
`climo` specifically you do need to reset the var list (notice `# NOTE: always overrides value in [default]` in `default.ini`), but I don't think that's the case with `ts`.

Reply:
Yes, I must have just had something set wrong at some point during testing. I'm able to generate the plots by only setting `vars` under `[default]`.

Reply:
It also works if I only specify under the `ts` subtasks, and not under `default` at all.